src/test/java/seedu/address/testutil/ItemUtil.java

```diff
@@ -20,14 +20,14 @@
      * Returns an add command string for adding the {@code item}.
      */
     public static String getAddCommand(Item item) {
-        return AddCommand.COMMAND_WORD + " " + getItemDetails(item);
+        return AddCommand.COMMAND_WORD + "|" + getItemDetails(item);
     }
 
     /**
      * Returns the part of command string for the given {@code person}'s details.
```

Comment:
Need to change person to item 😝

Reply:
I leave it up to the implementer to update the names of the methods and its corresponding javadocs.